data/hai/hai reveal 1 intro.txt

```diff
@@ -0,0 +1,947 @@
+		conversation
+			`You are directed to an official landing pad on your return to Hai-home and it is not long before you are introducing your entourage to Sayari. Alondo in particular seems quite stoked to be here and seems to have a knack for charming Sayari with polite and insightful, if rather simple, questions throughout the diplomatic niceties.`
+			`	You get down to business at the moment Sayeri inquires how your journey went.`
+			choice
+				`	"I don't recall there being nearly so many pirates."`
+			`	Sayari frowns and exhales a long breath through her nose, "Yes. Well it seems that the rumours of us, and the information making its way through your media has attracted every opportunist this side of the Deep."`
+			`	"Of course," she says, "some of the pirates were aware of us, and we were aware of them being aware of us. We had a quiet arrangement with a select few where we rewarded them for keeping the secret well contained within their own circles."`
+			`	Alondo looks surprised, but Mr Yang and Ms Adrianopolous both merely nod sagely.`
+			`	"Extra security forces are being called in, but we've been busier in Wah Ki than usual lately."`
+			`	Mr Yang pulls out a datapad, "Well then I'm afraid I may have some bad news. May I?"`
+			`	He gestures to a display and then takes control when indicated.`
+			`	"One of the things I brought with me was Republic Intelligence's estimate of the effectiveness of various information containment measures."`
+			`	A series of charts are thrown up on the screen. You're not an expert at reading this sort of data but highlighted percentages ranging from as low as 4% to only as high as 39% do not look promising. The final graphic is a map of stages of what is labelled as 'artifact penetration'.`
+			`	"Artifact penetration is the point at which a government cannot reasonably attempt to pretend something is other than the truth, at least not while still holding to the claim of being a representative government," explains Mr Yang.`
+			`	He checks the date, "Our most optimistic estimates suggest the Republic will be forced to acknowledge the existence of the Hai within three months or lose so much credibility as to compromise electoral stability, the actual time limit is probably sooner."`
+			choice
+				`	"Would that be so bad?"`
+			`	"It is always bad to be forced into acknowledging a secret, if we act sooner we may control the manner of that secret's release," says Yang.`
```

Comment:
It is worth noting that this is covered during the first contact with the Hai if the player bothers to ask questions. While it might be a bit of a reminder if people found the Hai long before this, it would be exceptionally repetitive if the player has followed the 'first time' route and then they brought it up again this swiftly.